.github/workflows/stale-pr.yml

```diff
@@ -0,0 +1,17 @@
+name: "Manage stale pull requests"
+on:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  - cron: "0 */6 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: "This pull request is stale because it has been open for 30 days with no activity. Please update to prevent this from being closed."
```

Comment:
Not sure if it could be good that the message is a bit friendlier, instead of being straight to the point (I guess this is copied from the action documentation).

Like explaining why we close stale PRs, letting the user know that they can always ping us later if they need it reopen...

Reply:
Good point, it does sound a bit blunt. I'll try to use softer wording; open to suggestions as well.